debian/control

```diff
@@ -102,6 +102,14 @@ Description: FRRouting suite - BGP RPKI support (rtrlib)
  number.
 Build-Profiles: <!pkg.frr.nortrlib>
 
+Package: frr-test-tools
```

Comment:
Wouldn't be good for RPM too?

Reply:
I spent 2 seconds looking at it and decided I have no idea how to do this in rpm land.  If someone needs this as part of the rpm stuff, then they can add it.  I feel it is ok for this approach because of the low usage of rpms